lib/puppet_x/vmware/vmware_lib/puppet_x/vmware/util.rb

```diff
@@ -0,0 +1,12 @@
+#
+# Wrapper to load the transport library from vmware_lib
```

Comment:
... and here it's the 'util library'. 

I know they are only comments, but if someone is depending on the comments for code this small, a little misdirection goes a long way into the bushes.